Bottom: cd47496d59283165f0d5c34d8f84cddd55d1c038
Top:    791459c1708fc05fce36bf9d7c24a818e6760291
Author: Christopher Voltz <cjunk@voltz.ws>
Date:   2017-05-01 23:25:10 -0500

Refresh of changelog

---

diff --git a/CHANGELOG.md b/CHANGELOG.md
index e7fee88..d727d01 100644
--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,6 +1,9 @@
 # Change Log
 
-## Unreleased
-### Added
-- tag, tag_row, and tag_col keywords which allow one to tag a cell in a
-  table and later get the row and column of the tagged cell
+## 2017-05-01
+- Added Rakefile to automate tasks used in creating the patchset.
+- Updated to 2017-02-09 upstream version of tablecalc.
+
+## 2016-04-29
+- Added tag, tag_row, and tag_col keywords which allow one to tag a cell in a
+  table and later get the row and column of the tagged cell.
